src/chrome/background.ts

```diff
@@ -1,44 +1,3 @@
+import 'webext-dynamic-content-scripts'
 import addDomainPermissionToggle from 'webext-domain-permission-toggle'
 addDomainPermissionToggle()
-
-async function inject(tabId: number) {
-  console.log('injecting...')
-  const { ChromeAdapter } = await import('./adapter')
-  new ChromeAdapter()
```

Comment:
I'm not sure about what this is doing and why it's being created at every injection. Should it be added as simply this?

```js
import {ChromeAdapter} from './adapter'

new ChromeAdapter()
```